Add Linux metadata files

DavidoTek opened the pull request:
This PR adds metadata files required for packaging on Linux, for example Flatpak. See https://github.com/mbrlabs/Lorien/issues/17.

The files are currently stored in [tinywrkb's Flatpak repo](https://github.com/tinywrkb/flathub/tree/com.github.mbrlabs.Lorien) but it is recommended to have the files in the upstream repository so they are the same for all packages and can easily be modified by the upstream author.

mbrlabs commented on public/linux/com.github.mbrlabs.Lorien.desktop:
```diff
@@ -0,0 +1,13 @@
+Name=Lorien
+GenericName=Note Taking
+Comment=Infinite canvas drawing/whiteboarding app. Made with Godot.
+Comment[de]=Unbegrenzte Zeichen- und Whiteboard-App. Erstellt mit Godot.
```
I'm gonna use German here since this a german translation.

Unbegrenzte Zeichen- und Whiteboard-App hört sich irgendwie komisch an. So als ob es normal ein "Zeichenlimit" geben würde. Ich bin mir aber auch gerade nicht sicher wie man das am besten übersetzt. Vieleicht einfach "Infinite Whitboarding App". Das ist zwar kein deutsch, sollten die meisten aber verstehen.

DavidoTek replied:
Alternativ einfach nur `Zeichen- und Whiteboard-App` oder `Freie Zeichen- und Whiteboard-App` (Frei im Sinne von kostenlos und unbegrenzt).

mbrlabs commented on public/linux/com.github.mbrlabs.Lorien.metainfo.xml:
```diff
@@ -0,0 +1,38 @@
+
+  <name>Lorien</name>
+  <summary>Infinite canvas drawing/whiteboarding app. Made with Godot.</summary>
+  <summary xml:lang="de_DE">Unbegrenzte Zeichen- und Whiteboard-App. Erstellt mit Godot.</summary>
```
Same

mbrlabs commented on public/linux/com.github.mbrlabs.Lorien.desktop:
```diff
@@ -0,0 +1,13 @@
+Type=Application
+Name=Lorien
+GenericName=Note Taking
+Comment=Infinite canvas drawing/whiteboarding app. Made with Godot.
```
Scratch the "Made with Godot". Irrelevant for the user.

mbrlabs reviewed (changes requested):
Thanks for the PR and sorry for the delay in review! Basically just some german translation issues.

DavidoTek commented:
I have changed the german description to `Freie Zeichen- und Whiteboard-App` and changed the mime type to `x-lorien-canvas`.

If there are no other changes, I can squash the commits.
